.goreleaser.yml

```diff
@@ -50,10 +50,39 @@ changelog:
 
 dockers:
   - dockerfile: build/Dockerfile
+    use: buildx
+    goos: linux
+    goarch: amd64
+    build_flag_templates:
+      - "--platform=linux/amd64"
     image_templates:
-      - "getwoke/woke:latest"
-      - "getwoke/woke:{{ .Major }}.{{ .Minor }}"
-      - "getwoke/woke:{{ .RawVersion }}"
+      - "getwoke/woke:latest-amd64"
+      - "getwoke/woke:{{ .Major }}.{{ .Minor }}-amd64"
+      - "getwoke/woke:{{ .RawVersion }}-amd64"
+  - dockerfile: build/Dockerfile
+    use: buildx
```

Comment:
Yeah, well, the thing it's that it should create the manifest, otherwise we can use this way: https://github.com/cloudnative-pg/cloudnative-pg/blob/main/.github/workflows/release-publish.yml#L157 which basically use the `build-push-action` to create the image but it's not using the GoReleaser way.

If you prefer, I can go with this approach instead of using GoReleaser ot create the image, just let me know! BTW it's much easier to understand and maintain in my opinion.

Reply:
I'd prefer to stick with goreleaser, I think it's probably fine if `--snapshot` doesn't exercise the `docker manifest` section. I just wanted to note it so we're aware. It's more of a nuance of Docker and less with goreleaser

Reply:
Goreleaser is run via a github action, so you might still need to update that in order for buildx to work - https://github.com/docker/setup-buildx-action (if you merge in main, you'll get a new PR check that might help confirm things are working)

Reply:
@caitlinelfring I'll update and add the buildx support then :D just give me a couple of minutes

Reply:
@caitlinelfring there you go! the setup-buildx-action was added and with the qemu in case we want to add more platforms in the future, probably adding arm/v7 will be a good idea, what do you thinik?

Reply:
I think this is good for now. One last comment and I'm good to approve this PR.

I'll probably trigger a release because I just merged a bunch of dependency upgrades. Then I'll merge this and trigger a new release